client/src/Components/UserInfoCard/index.tsx

```diff
@@ -26,8 +31,27 @@ function UserInfoCard({ userInfo }: UserInfoCardProps) {
     },
     totalReviews,
   } = userInfo;
-
   const { t } = useTranslation();
+  const [visible, setVisible] = useState(false);
+  const onCreate = async (values: any) => {
+    try {
+      setVisible(false);
+      const formData = { ...values, userId: id };
+      console.log(formData);
+      const response = await axios.post('/api/v1/reviews', formData);
+      message.success(t('review-message'));
+      console.log(response);
+    } catch (error:any) {
+      if (error.response.status === 400) {
+        setVisible(true);
+        message.error(t('review-error'));
+        console.log(error.response.data.msg);
+      } else if (error.response.status === 500) {
+        setVisible(true);
+        message.error(t('review-error'));
```

Comment:
Did you notice that the behaviour of error handling is annoying,
 First modal closed then you display unclear message then you open the modal Again!!
Why does this
Instead message.error(error.response.data.msg), make sure all messages translated in locales files,
Then don't open the modal again
```suggestion
    
        message.error(t(error.response.data.msg));
```